spacy/tests/test_language.py

```diff
@@ -550,6 +550,7 @@ def test_spacy_blank():
         ("en", "en"),
         ("fra", "fr"),
         ("fre", "fr"),
+        ("isl", "is"),
```

Comment:
This one needs to be reversed
```suggestion
        ("is", "isl"),
```